packages/flutter/test/widgets/page_view_test.dart

```diff
@@ -1096,4 +1096,30 @@ void main() {
 
     expect(tester.widget<SliverFillViewport>(viewportFinder()).padEnds, false);
   });
+
+  testWidgets('PageView - precision error inside RenderSliverFixedExtentBoxAdaptor', (WidgetTester tester) async {
+    // Regression test for https://github.com/flutter/flutter/issues/95101
+
+    final PageController controller = PageController(initialPage: 152);
+    await tester.pumpWidget(
+      Center(
+        child: SizedBox(
+          width: 392.72727272727275,
+          child: Directionality(
+            textDirection: TextDirection.ltr,
+            child: PageView.builder(
+              controller: controller,
+              itemCount: 366,
+              itemBuilder: (BuildContext context, int index) {
+                return const SizedBox();
+              },
+            ),
+          ),
+        ),
+      ),
+    );
+
+    controller.jumpToPage(365);
+    await tester.pump();
```

Comment:
The test should expect for something

maybe add an `expect(tester.takeException(), isNull)`